cmd/entrypoint-wrapper/main.go

```diff
@@ -298,12 +299,12 @@ func manageHome(proc *exec.Cmd) {
 // manageKubeconfig provides a unique writeable kubeconfig so users can for example set a namespace,
 // but changes are not propagated to subsequent steps to limit the amount of possible mistakes
 func manageKubeconfig(proc *exec.Cmd) error {
+	tempPrefix := "temp-"
 	if original, set := os.LookupEnv("KUBECONFIG"); set {
-		writableCopy, err := ioutil.TempFile("", "kubeconfig-*")
+		writableCopy, err := ioutil.TempFile("", tempPrefix+"kubeconfig-*")
 		if err != nil {
 			return fmt.Errorf("could not create unique, writeable $KUBECONFIG copy: %w", err)
 		}
-		proc.Env = append(proc.Env, fmt.Sprintf("KUBECONFIG=%s", writableCopy.Name()))
```

Comment:
I'm a little bit perplexed on unsetting `$KUBECONFIG` at the beginning, but I may lack of context. What if tests rely on having an even empty `$KUBECONFIG` as soon as they start? Let's suppose a test does what follows:
- it starts and check if `$KUBECONFIG` exists (otherwise failing), without caring if it is actually empty and/or writable and/or valid, etc. 
- it does its stuff
- then it uses the `$KUBECONFIG` and at this moment it is (hopefully) non-empty and valid  

Such a test would fail if we introduce those changes. Again, I'm not even sure that situation would be possible but it's worth discussing.

Reply:
@jmguzik just ignore my comment for now as it may not have any sense at all

Reply:
it is being discussed on our slack channel